src/uwtools/drivers/cdeps.py

```diff
@@ -2,74 +2,39 @@
 A driver for the CDEPS data models.
 """
 
-from datetime import datetime
 from pathlib import Path
-from shutil import copy
-from typing import Optional
 
-from iotaa import asset, dryrun, task, tasks
+from iotaa import asset, task, tasks
 
-from uwtools.api.template import render
+from uwtools.api.template import _render
 from uwtools.config.formats.nml import NMLConfig
-from uwtools.drivers.driver import Driver
-from uwtools.logging import log
+from uwtools.drivers.driver import AssetsCycleBased
 from uwtools.strings import STR
-from uwtools.utils.tasks import file, filecopy, symlink
+from uwtools.utils.tasks import file
 
 
-class CDEPS(Driver):
+class CDEPS(AssetsCycleBased):
```

Comment:
On `uwtools` `main`, we recently introduced several more-precise driver classes that provide correct `__init__()` and `_taskname()` methods, so those no longer need to be defined in this class.